File: app/src/main/java/dk/itu/moapd/realmdb/ActorName.java
package dk.itu.moapd.realmdb;

import androidx.annotation.NonNull;
import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

public class ActorName extends RealmObject {

    @PrimaryKey
    private int id;

    @Required
    private String name;

    public ActorName() {
    }

    public ActorName(int id, @NonNull String name) {
        this.id = id;
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @NonNull
    public String getName() {
        return name;
    }

    public void setName(@NonNull String name) {
        this.name = name;
    }

}

File: app/src/main/java/dk/itu/moapd/realmdb/ActorName.kt
package dk.itu.moapd.realmdb

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.Required

open class ActorName(
    @PrimaryKey var id: Int = 0,
    @Required var name: String = ""
) : RealmObject()
